dfx-library/xcode/dfxplugin.xcconfig

```diff
@@ -6,7 +6,7 @@ DEPLOYMENT_LOCATION = YES
 DSTROOT = /
 SKIP_INSTALL = NO
 GCC_C_LANGUAGE_STANDARD = c99
-CLANG_CXX_LANGUAGE_STANDARD = c++17
+CLANG_CXX_LANGUAGE_STANDARD = c++2a
```

Comment:
Probably we should keep this in sync across the platforms, but FYI I did not change this in my push

Reply:
agreed I will undo for Xcode as well.